components/Voting/EloVote.styled.tsx

```diff
@@ -63,19 +68,30 @@ export const StyledSessionAbstract = styled('div')(({ theme }) => ({
 }))
 
 type StyledVoteButtonProps = {
-  kind: 'primary' | 'secondary'
+  kind: 'primary' | 'secondary' | 'tertiary'
 }
 
+export const StyledEloButtonContainer = styled('div')(() => ({
+  display: 'flex',
+  flexFlow: 'row',
+  justifyContent: 'center',
+  alignItems: 'center',
+}))
+
 export const StyledVoteButton = styled(Button)<StyledVoteButtonProps>(({ theme, kind }) => ({
   alignSelf: 'center',
-  textTransform: 'uppercase',
-  borderRadius: calcRem(4),
+  //textTransform: 'uppercase',
+  borderRadius: `${kind == 'primary' ? calcRem(4) : 0} ${kind == 'secondary' ? calcRem(4) : 0}  ${
+    kind == 'secondary' ? calcRem(4) : 0
+  } ${kind == 'primary' ? calcRem(4) : 0}`,
```

Comment:
Seems strange to do this based on the `kind` prop of the "primary". You could add a `position` prop and change this based on the position (see the suggestions I've added for the diff, not any actual obligation to change)

Alternatively, might be easier to group them slightly differently so all the "left", "middle" and "right" corner styles are together, e.g.

```suggestion
  borderRadius: kind == 'primary' ? `${calcRem(4)} 0 0 ${calcRem(4)}` : kind === 'secondary' ? `0 ${calcRem(4)} ${calcRem(4)} 0` : 0,
```